modules/log4cats/src/main/scala/doobie/log4cats/Log4CatsLogHandler.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2013-2020 Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package doobie.log4cats
+
+import doobie.LogHandler
+import doobie.util.log._
+import org.typelevel.log4cats._
+
+class Log4CatsLogHandler[F[_]](logger: MessageLogger[F]) extends LogHandler[F]{
+  override def run(logEvent: LogEvent): F[Unit] = logEvent match {
+    case Success(s, a, l, e1, e2) =>
+      logger.info(
+        s"""Successful Statement Execution:
+           |
+           |  ${s.linesIterator.dropWhile(_.trim.isEmpty).mkString("\n  ")}
+           |
+           | arguments = [${a.mkString(", ")}]
+           | label     = $l
+           |   elapsed = ${e1.toMillis.toString} ms exec + ${e2.toMillis.toString} ms processing (${(e1 + e2).toMillis.toString} ms total)
+        """.stripMargin)
+
+    case ProcessingFailure(s, a, l, e1, e2, t) =>
+      logger.warn(
```

Comment:
I think this should be an error too? (e.g. error in Put/Get definition)

Reply:
It seemed to me not as severe like `ExecFailure`, and giving it a different priority also give more power to tune what (and how) is logged.

But if you'd still prefer to have both as errors, no problem, just let me know.

Reply:
Thinking about it more, perhaps both these failure cases should be `warn` level because 

- If the failure isn't intentional, their code is going to fail anyway and their execution will fail with an Exception anwyay. 
- If the (Execution) failure is intentional, then we don't really want to log at error level to avoid triggering alerts.

WDYT? I don't really mind